__tests__/app.test.js

```diff
@@ -0,0 +1,38 @@
+const request = require("supertest");
+const app = require("../db/app");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const data = require("../db/data/test-data");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("app/topics", () => {
+  test("200: returns an array of topic objects with slug and description properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        //  console.log(body);
+        expect(body).toBeInstanceOf(Array);
+      });
+  });
+  test("200: the returning array is made up of objects and each one has a slug and description keys ", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((object) => {
```

Comment:
Include a check for the length of the array, 0 length array will automatically pass and we want the length to be what we expect.